components/stepper/index.css

```diff
@@ -315,3 +283,33 @@ governing permissions and limitations under the License.
   border-end-end-radius: var(--mod-stepper-border-radius, var(--spectrum-stepper-border-radius));
   border-inline-end-width: var(--mod-stepper-border-width, var(--spectrum-stepper-border-width));
 }
+
+@media (forced-colors: active) {
+  .spectrum-Stepper {
+    --highcontrast-stepper-border-color: CanvasText;
+    --highcontrast-stepper-border-color-hover: Highlight;
+    --highcontrast-stepper-border-color-focus: Highlight;
+    --highcontrast-stepper-border-color-focus-hover: Highlight;
+    --highcontrast-stepper-border-color-keyboard-focus: CanvasText;
+
+    &.is-disabled {
+      --highcontrast-stepper-border-color: GrayText;
+      --highcontrast-stepper-buttons-border-width: var(--mod-stepper-border-width, var(--spectrum-stepper-border-width));
+    }
+
+    &.is-invalid {
+      --highcontrast-stepper-border-color: Highlight;
+      --highcontrast-stepper-border-color-hover: Highlight;
+      --highcontrast-stepper-border-color-focus: Highlight;
+      --highcontrast-stepper-border-color-focus-hover: Highlight;
+      --highcontrast-stepper-border-color-keyboard-focus: Highlight;
+    }
+
+    --highcontrast-stepper-button-background-color-default: Canvas;
+    --highcontrast-stepper-button-background-color-hover: Canvas;
+    --highcontrast-stepper-button-background-color-focus: Canvas;
+    --highcontrast-stepper-button-background-color-keyboard-focus: Canvas;
+
+    --highcontrast-stepper-focus-indicator-color: Highlight;
```

Comment:
Can we reorder these to above the selectors with the other root-level variables? [minor request]

Reply:
reordered!